### PR TITLE
nicer messages for empty teams and invitations table

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

**Comment by lorenzleutgeb** [comment]
Please check in binary files with Git LFS.

**Comment by pbochis** [comment]
http://prntscr.com/bdv8kr 
http://prntscr.com/bdv8qu

File: app/elements/coder/team/user-team-invitations-table.html [hunk 1]
```diff
@@ -23,26 +46,34 @@
...
+						<paper-icon-button icon="close" on-click="_delete"></paper-icon-button>
+					</template>
+				</paper-datatable-column>
+			</paper-datatable>
+		</template>
+		<template is="dom-if" if="[[!hasInvitations]]">
+			<iron-image src="/images/invite.png" sizing="contain" class="placeholder"></iron-image>
+			<div id="messageWrapper">
+				<span></span>
+				<div>You'll be notified here when someone invites you into a team</div>
```
Comment (lorenzleutgeb) [inline app/elements/coder/team/user-team-invitations-table.html]: "... invites you _to join_ a team**.**"

**Comment by pbochis** [comment]
With more margin 
http://prntscr.com/bdvbj3

**Pull request merged**